lib/tasks/sitemap.rake

```diff
@@ -0,0 +1,139 @@
+namespace :sitemap do
```

Comment:
Let's put this within the `fromthepage` namespace, so that the task can be run with `rake fromthepage:sitemap:generate`